.github/workflows/build-workflow.yml

```diff
@@ -58,6 +57,30 @@ jobs:
           name: debian-packages-${{ matrix.job.target }}
           path: target/${{ matrix.job.target }}/debian/*.deb
 
+  publish:
+    name: Publish ${{ matrix.job.target }}
+    runs-on: Ubuntu-20.04
+    needs: [build]
+    strategy:
+      matrix:
+        job:
+          - { target: x86_64-unknown-linux-musl,      repo: tedge-main,          component: main }
+          - { target: aarch64-unknown-linux-musl,     repo: tedge-main,          component: main }
+          # Keep arm-unknown-linux-gnueabihf in separate repo due to armhf conflict between raspbian and debian
```

Comment:
Does it mean that `arm-unknown-linux-muslabihf`?

Reply:
Fixed by d6864228e976037a83ab7b5bdf36743f686932aa